File: src/targets/php/guzzle/fixtures/application-json.php
<?php
<<<<<<< HEAD
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
=======

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
>>>>>>> upstream/master
  'body' => '{"number":1,"string":"f\\"oo","arr":[1,2,3],"nested":{"a":"b"},"arr_mix":[1,"a",{"arr_mix_nested":{}}],"boolean":false}',
  'headers' => [
    'content-type' => 'application/json',
  ],
]);

echo $response->getBody();

File: src/targets/clojure/clj_http/fixtures/application-json.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/post "https://httpbin.org/anything" {:content-type :json
                                             :form-params {:number 1
                                                           :string "f\"oo"
                                                           :arr [1 2 3]
                                                           :nested {:a "b"}
                                                           :arr_mix [1 "a" {:arr_mix_nested {}}]
                                                           :boolean false}})
=======
(client/post "http://mockbin.com/har" {:content-type :json
                                       :form-params {:number 1
                                                     :string "f\"oo"
                                                     :arr [1 2 3]
                                                     :nested {:a "b"}
                                                     :arr_mix [1 "a" {:arr_mix_nested {}}]
                                                     :boolean false}})
>>>>>>> upstream/master
